### ISSUE TITLE
Support multiple children

### ISSUE THREAD

**Issue by zieren** [zieren/eltern-emailer#12]
It should be straightforward to retrieve the list of students and select each one in turn.

**Comment by zieren** [comment]
We may also consider individual names for kids, possibly replacing names for schools.

**Comment by pimisen** [comment]
crucial for me .... hope it will be availiable soon. ics file per kid with Test and Schulaufgaben (Event-Classes event-warning and event-alert are used for that at our school)

**Comment by zieren** [comment]
@pimisen Would you be able to help with this? I simply don't have a test case. Could you save a page (e.g. the landing page) where the selector is shown and [share it with me](https://zieren.de/contact)?

**Comment by pimisen** [comment]
Hi Joerg, @zieren sure, how can i send you copy of the complete Website (tar.gz.)? The contact - form does not offer a file-attachement.....

**Comment by zieren** [comment]
I was thinking of a Google Drive link or something, but that code above should be fine already.

I assume after login you select the kid and that selection remains constant during the entire session until you either log out or select the other kid?

**Comment by zieren** [comment]
Actually, it would be great if you could get in touch via the contact form (so we exchange emails) because there might be more to discuss. Hope you're ok with that, otherwise we can also continue here.

**Comment by pimisen** [comment]
> I was thinking of a Google Drive link or something, but that code above should be fine already.
> 
> I assume after login you select the kid and that selection remains constant during the entire session until you either log out or select the other kid?

Yes, exactly it is persistent during the session or until the selection of another kid. I wrote you via the Contact Form :-) Thanks in advance!
